items/blind.lua

```diff
@@ -675,6 +675,40 @@ local scorch = {
 		end
 	end,
 }
+-- +0.25X blind requirements
+-- for every $5 you have when selected
+local greed = {
+	dependencies = {
+		items = {
+			"set_cry_blind",
+		},
+	},
+	object_type = "Blind",
+	name = "cry-greed",
+	key = "greed",
+	pos = { x = 0, y = 0 }, -- use Tax as placeholder icon
+	boss = {
+		min = 1,
+		max = 10,
+	},
+	atlas = "blinds",
+	order = 22,
+	boss_colour = HEX("4ca180"),
+	loc_vars = function(self, info_queue, card)
+		return { vars = { (get_blind_amount(G.GAME.round_resets.ante) * 0.25) } }
```

Comment:
Use the values from the extra/immutable table when passing into the localization, and support BigNums

```suggestion
		return {
		    vars = {
		        number_format(card.ability.extra.money_factor),
		        number_format(lenient_bignum((get_blind_amount(G.GAME.round_resets.ante) * to_big(card.ability.extra.blind_mod))),
		    }
		}
```

Reply:
If we cant get rid of magic numbers then: 
```suggestion
		return {
		    vars = {
		        5,
		        number_format(get_blind_amount(G.GAME.round_resets.ante) * 0.25),
		    }
		}
```